packages/storybook/stories/snippets/App.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
```

Comment:
It is OK that this file isn't named `*.snippet.tsx` because it gets imported by `ReactIndex.snippet.tsx` and gets tested that way.